core-build/gulp-core-build-sass/src/sass.schema.json

```diff
@@ -23,14 +23,14 @@
     },
 
     "sassMatch": {
-      "title": "Sass File Glob Pattern",
+      "title": "SASS File Glob Pattern",
```

Comment:
This should probably go back to “Sass”. My mistake! (Always thought it was capitalized, as it’s an acronym. Nope!)